src/components/CreateBlock.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
```

Comment:
Write down a sample usage in the comment here, so that others can understand how to use it. Comment down an example.